sdk/storage/azure-storage-queue/azure/storage/queue/_shared/uploads_async.py

```diff
@@ -296,7 +296,7 @@ async def _upload_chunk(self, chunk_offset, chunk_data):
 
     async def _upload_substream_block(self, index, block_stream):
         try:
-            block_id = f'BlockId{index/self.chunk_size:05d}'
```

Comment:
I'm surprised this is working at the moment?? I guess it doesn't matter since it just changes the resultant `block_id`, but the random contributor did not give a properly equivalent statement to the original format string: `block_id = 'BlockId{}'.format("%05d" % (index/self.chunk_size))`

Sample Code:
```index = 10
chunk_size = 5

block_id = 'BlockId{}'.format("%05d" % (index/chunk_size))
block_id2 = f'BlockId{index/chunk_size:05%}'
block_id3 = f'BlockId{"%05d" % (index/chunk_size)}'

print('Original format:', block_id)
print("F-string from random contributor:", block_id2)
print("Fixed f-string:", block_id3)
```

Sample Output:
```
Original format: BlockId00002
F-string from random contributor: BlockId200.000000%
Fixed f-string: BlockId00002
```

Reply:
Wowza, good catch.